substrate-chainlink/docker-compose.yml

```diff
@@ -18,11 +18,11 @@ x-postgres-def:
     - "POSTGRES_HOST_AUTH_METHOD=trust"
 x-external-initiator-def:
   &external-initiator-def
-  image: smartcontract/external-initiator
+  image: laurenttrk/external-initiator:2.0.0
   command: '{\"name\":\"substrate-node\",\"type\":\"substrate\",\"url\":\"${SA_ENDPOINT}\"}'
 x-substrate-adapter-def:
   &substrate-adapter-def
-  image: smartcontract/substrate-adapter
+  image: laurenttrk/substrate-adapter:2.0.0
```

Comment:
Just putting a reminder here to update these. Ignore for now (but please don't resolve)

Reply:
Should be good to change back now. The adapter tag is `smartcontract/substrate-adapter:substrate-2.0.0`

Reply:
Great ! Do you have a tag for the external initiator as well ?

Reply:
`smartcontract/external-initiator:latest` should be fine

Reply:
Ok, I just updated the version :)